huaweicloud/services/iec/data_source_huaweicloud_iec_flavors.go

```diff
@@ -90,18 +91,18 @@ func dataSourceIecFlavorsV1Read(d *schema.ResourceData, meta interface{}) error
 		Operator: d.Get("operator").(string),
 	}
 
-	logp.Printf("[DEBUG] fetching IEC flavors by filter: %#v", listOpts)
+	log.Printf("[DEBUG] fetching IEC flavors by filter: %#v", listOpts)
 	allFlavors, err := flavors.List(iecClient, listOpts).Extract()
 	if err != nil {
-		return fmtp.Errorf("Unable to extract iec flavors: %s", err)
+		return fmt.Errorf("unable to extract IEC flavors: %s", err)
 	}
 	total := len(allFlavors.Flavors)
 	if total < 1 {
-		return fmtp.Errorf("Your query returned no results of huaweicloud_iec_flavors. " +
-			"Please change your search criteria and try again.")
+		return fmt.Errorf("your query returned no results of iec_flavors," +
+			"please change your search criteria and try again")
```

Comment:
please add an space at the end of `,` and check it all.
```suggestion
		return fmt.Errorf("your query returned no results of iec_flavors, " +
			"please change your search criteria and try again")
```